Intersect.Client/Entities/Player.cs

```diff
@@ -2406,9 +2409,45 @@ public override void DrawName(Color textColor, Color borderColor, Color backgrou
                 }
             }
 
+            if (Globals.Me.Id != Id)
+            {
+                // Party member names
+                if (Globals.Me.IsInMyParty(this) && CustomColors.Names.Players.TryGetValue("PartyMember", out var partyColors))
+                {
+                    textColor = partyColors.Name;
+                    borderColor = partyColors.Outline;
+                    backgroundColor = partyColors.Background;
+                }
+                // Guildies
+                else if (Globals.Me.IsInGuild && Guild == Globals.Me.Guild && CustomColors.Names.Players.TryGetValue("GuildMember", out var guildColors))
+                {
+                    textColor = guildColors.Name;
+                    borderColor = guildColors.Outline;
+                    backgroundColor = guildColors.Background;
+                }
+
+                // Enemies in PvP
+                if (!Globals.Me.IsAllyOf(this) && Globals.Me.MapInstance.ZoneType != MapZone.Safe && CustomColors.Names.Players.TryGetValue("Hostile", out var hostileColors))
```

Comment:
Is there a reason this isn't `else if`d?

Reply:
My reasoning is that in the _future_, if there are additional rules about what constitutes an ally (i.e, maybe you can kill guildies in an Arena map), then we won't miss this. Basically, if someone is hostile, their name color will _always_ be overridden.